rules/ai-tools-voice-translations/rule.md

```diff
@@ -8,30 +8,32 @@ authors:
 created: 2023-11-10T03:23:18.710Z
 guid: 11393539-d0b4-4c61-bf69-7aa98ecb5e36
 ---
-AI tools like[ Rask](https://app.rask.ai/auth), [Voice.ai](https://voice.ai/home), and [ElevenLabs](https://elevenlabs.io) are revolutionizing how we generate voices and translations. They can even create transcripts, making our lives a lot easier. But remember, with great power comes great responsibility! 🦸‍♂️
+AI tools like[Rask](https://app.rask.ai/auth), [Voice.ai](https://voice.ai/home), and [ElevenLabs](https://elevenlabs.io) are revolutionizing how we generate voices and translations. They can even create transcripts, making our lives a lot easier. But remember, with great power comes great responsibility!
 
 <!--endintro-->
 
+::: bad
+
+![Figure: Bad Example - Relying solely on AI without any oversight can lead to errors.](rask-bad-v2.png)
+
+:::
+
 ::: good
 
 ![Figure: Good Example - Using Rask to translate an English voiceover into French in the speakers own voice.](rask-good-v2.png)
 
-Video -  [Fix Your Website Chatbot! - English version](https://youtube.com/shorts/vSXy1VjS_b4)
+`youtube: https://www.youtube.com/embed/vSXy1VjS_b4`
+
+Video -  Fix Your Website Chatbot! - English version (1min)
```

Comment:
Hey @MarcusIrmscherSSW 

- [ ] 1. I think this works best as "Video: " rather than with the dash. Please change here and in the French one
- [ ] 2. Make the caption for the video in bold